src/Console/Command/SelfUpdateCommand.php

```diff
@@ -123,36 +136,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $tempFilename = basename($localFilename, '.phar').'-tmp.phar';
-        $remoteFilename = sprintf('https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/%s/php-cs-fixer.phar', $remoteTag);
+        $tempFilename = dirname($localFilename).'/'.basename($localFilename, '.phar').'-tmp.phar';
+        $remoteFilename = $this->toolInfo->getPharDownloadUri($remoteTag);
 
-        try {
-            $copyResult = @copy($remoteFilename, $tempFilename);
-            if (false === $copyResult) {
-                $output->writeln(sprintf('<error>Unable to download new version %s from the server.</error>', $remoteTag));
-
-                return 1;
-            }
+        $copyResult = @copy($remoteFilename, $tempFilename);
```

Comment:
no need for variable introduction, it is only used one time in the `if` below

Reply:
Variable removed.